src/20240527_AaveV3Base_WeETHAaveV3BaseOnboarding/WeETHAaveV3BaseOnboarding.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "weETH Aave V3 Base Onboarding"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-onboarding-of-weeth-to-aave-v3-on-base/17691"
+---
+
+## Simple Summary
+
+This ARFC seeks to add [Ether.fi](http://ether.fi/) Liquid Restaking Token weETH to Aave V3 on Base.
+
+After the successful onboarding of both weETH to [Aave v3 Ethereum](https://vote.onaave.com/proposal/?proposalId=74&ipfsHash=0x227ef8b0f49775f64100ec697bc4e67b0739bd1ff08788b1f6b48a66e1d57bf7) and [Arbitrum](https://vote.onaave.com/proposal/?proposalId=76&ipfsHash=0x78778591515790b337fcdcc2a02d49dc58e98cad614c33d61e1173bc6194729d), and latest ARFC to Increase Supply and Borrow Caps for weETH on both networks due to huge demand, this proposal will go direct-to-AIP following the asset onboarding framework.
+
+## Motivation
+
+eETH is an LRT that allows users to stake their ETH, accrue staking rewards, and receive additional rewards through native restaking on EigenLayer.
+
+[Ether.fi](http://ether.fi/) has also launched eETH on Arbitrum, allowing users on the L2 to get exposure to the LRT yield and points. As weETH has already been approved for onboarding to Aave v3 Ethereum and Arbitrum by the DAO, this proposal aims to extend the onboarding of weETH to Aave v3 on Base.
+
+## Specification
+
+The table below illustrates the configured risk parameters for **weETH**
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
```

Comment:
should be false here

Reply:
fixed.

Interesting same yarn generate issue happened but wasn't detected for https://github.com/bgd-labs/aave-proposals-v3/pull/266/files#diff-e764ceeec86af20be4f5b784f3c72458aa392de79ab962ed9958eff99849fa4fR42

@sakulstra